include/util/static_rtree.hpp

```diff
@@ -432,11 +418,6 @@ class StaticRTree
         std::priority_queue<QueryCandidate> traversal_queue;
         traversal_queue.push(QueryCandidate{0, TreeIndex{0}});
 
-        // we need to limit the size of this vector to MAX_NUM_CANDIDATES
```

Comment:
I have removed caching, it was the reason for failing tests with "tiny" r-trees. With mapped files only a pair of indices and fixed_projected_coordinate is needed in QueryCandidate without full copy of EdgeData.